votepy/testing/algorithms/base_algorithm.py

```diff
@@ -0,0 +1,44 @@
+from votepy.ordinal_election import OrdinalElection
```

Comment:
Should all these files reside under testing?

Reply:
No, I will move them one level up in the files' hierarchy and update imports